taskfiles/boost/boost-test.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <string>
+
+#include <boost/any/bad_any_cast.hpp>
+#include <boost/program_options/errors.hpp>
+#include <boost/program_options/options_description.hpp>
+#include <boost/program_options/parsers.hpp>
+#include <boost/program_options/value_semantic.hpp>
+#include <boost/program_options/variables_map.hpp>
+
+namespace {
+auto parse_args(int argc, char** argv) -> boost::program_options::variables_map {
+    boost::program_options::options_description desc;
+    desc.add_options()("help", "Boost task test. Check the string against its size.");
+    desc.add_options()("input", boost::program_options::value<std::string>(), "Input string.");
+    desc.add_options()("size", boost::program_options::value<int>(), "Size of the string.");
+    boost::program_options::variables_map variables;
+    boost::program_options::store(
+            // NOLINTNEXTLINE(misc-include-cleaner)
```

Comment:
Is there really no way to properly include `boost::program_options::parse_command_line`?